tests/e2e/device/config/core_mqtt_config.h

```diff
@@ -0,0 +1,78 @@
+/*

```

Comment:
Similar to my question about freeRtosConfig.h above, could this / should this be in a tests/common folder??

Reply:
updated